plugins/ibm_qradar/plugin.spec.yaml

```
@@ -0,0 +1,982 @@
+      required: false
+    remote_destination_count:
+      title: "Remote Destination Count"
+      type: integer
+      description: "Remote destination count"
+      required: false
+    rules:
+      title: "Rules"
+      type: "[]rules"
+      description: "Rules"
+      required: false
+    security_category_count:
+      title: "Security Category Count"
+      type: integer
+      description: "Security category count"
+      required: false
+    severity:
+      title: "Severity"
+      type: integer
+      description: "Severity"
+      required: false
+    source_address_ids:
+      title: "Source Address Ids"
+      type: "[]integer"
+      description: "Source address ids"
+      required: false
+    source_count:
+      title: "Source Count"
+      type: integer
+      description: "Source count"
+      required: false
+    source_network:
+      title: "Source Network"
+      type: string
+      description: "Source network"
+      required: false
+    start_time:
+      title: "Start Time"
+      type: integer
+      description: "Start time"
+      required: false
+    status:
+      title: "Status"
+      type: string
+      description: "Status"
+      required: false
+    username_count:
+      title: "Username Count"
+      type: integer
+      description: "Username count"
+      required: false
+actions:
+  start_ariel_search:
+    title: Start Ariel Search
+    description: Start Ariel Search
+    input:
+      aql:
+        title: AQL Query
+        type: string
+        description: AQL query to perform the search
+        required: true
+        example: Select * from events
+    output:
+      data:
+        title: Search Data
+        type: search
+        description: JSON Data of the Search
+        required: false
+  get_ariel_search_by_id:
+    title: Get Ariel Search by ID
+    description: Get ariel search by ID
+    input:
+      search_id:
+        type: string
+        required: true
+        description: Specific Ariel Search Id for which the search
+        example: 9102cb1d-5994-4f78-8f08-16c6f6991015
+      poll_interval:
+        type: number
+        required: false
+        description: Poll interval is the number of seconds to recheck until the search gets COMPLETED
+        default: 1
+        example: 1
+    output:
+      data:
+        title: Search Data
+        type: search
+        description: JSON Data of the Search
+        required: false
+  get_assets:
+    title: Get Assets
+    description: List all assets found in the model
+    input:
+      range:
+        type: string
+        required: false
+        description: Use this parameter to restrict the number of elements that are returned in the list to a specified range. The list is indexed starting at zero
+        example: 1-2
+      filter:
+        type: string
+        required: false
+        description: This parameter is used to restrict the elements in a list base on the contents of various fields
```

```suggestion
        description: Restrict the elements in a list based on the contents of various fields
```
Can the user input multiple criteria here? If so, what would the format be?